exercises/practice/prime-factors/prime-factors.rakutest

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use PrimeFactors;
+
+cmp-ok( # begin: 924fc966-a8f5-4288-82f2-6b9224819ccd
+    factors(1),
+    "eq",
```

Comment:
`~~` or `eqv` for list comparison.

Reply:
I guess I'm looking for a type-agnostic comparator (`cmp`), so that we can be accepting of solutions returning different forms of the same thing but I'm unsure how to go about that or if it is advised ?